bin/perlbug

```diff
@@ -2,7 +2,7 @@
     eval 'exec /home/git/binary-com/perl/bin/perl -S $0 ${1+"$@"}'
 	if $running_under_some_shell;
 
-my $config_tag1 = '5.26.2 - Mon Jul 30 11:50:48 UTC 2018';
+my $config_tag1 = '5.26.2 - Mon Feb  3 08:21:42 UTC 2020';
```

Comment:
I've no idea how this all works, but why is the version and patchlevel_date staying the same, but the date changes?

Reply:
This is filled in by the `make`s during build/install, presumably to let one know how recent this perl was configured.